history-service/server.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const { Pool } = require('pg');
+const cors = require('cors');
+
+const service_port = 5433;
+const sql_port = 5432;
+
+const app = express();
+const pool = new Pool({
+    user: 'postgres-db',
+    host: 'postgres-db9000',
+    database: 'submission-db',
+    password: 'postgres-db',
+    port: `${sql_port}`,
+});
+
+// Middlewares
+app.use(cors());
+app.use(express.json());
+
+// Get submission history of a user
+app.get('/user/:username', async (req, res) => {
+    try {
```

Comment:
I think it would be good to organise this into the MVC(S) pattern! The question service has a decent example of this